nodewatcher/modules/frontend/api/templates/rest_framework/base.html

```diff
@@ -1,21 +1,230 @@
-{% extends "rest_framework/base.html" %}
```

Comment:
From my understanding this required a fork of DRF. I don't like that, so I changed it in a way that the fork was not required and bumping versions is easier. Unfortunately it seems like they didn't agree with your proposed changes which would make it more modular ;-)